fairscale/nn/data_parallel/oss_ddp.py

```diff
@@ -99,6 +99,15 @@ def __getstate__(self) -> Dict:
         attrs = copy.copy(self.__dict__)
         return attrs
 
+    def train(self, mode: bool = True) -> "OssDdp":
```

Comment:
Good question. I thought it was mypy needed it, but it is python parser that needed it because this is like a forward declaration. It is used inside the definition of the class itself. Otherwise, I get an error like:

ERROR tests/nn/data_parallel/test_oss_ddp.py - NameError: name 'OssDdp' is not defined

Reply:
Ah. Didn't know that. Found a nice description here:

https://stackoverflow.com/questions/33533148/how-do-i-specify-that-the-return-type-of-a-method-is-the-same-as-the-class-itsel

Reply:
Ah, that's good answer you found. Should I change it and assume py 3.7+? Or leave it as is?